lib/menu.rb

```diff
@@ -0,0 +1,7 @@
+class Menu
+
+def dishes
+{"Roast Pork Belly Hirata Buns" => 5, "Salt & Pepper Squid" => 5, "Sesame Glazed Squid" => 5, "Char Siu Pork Bao" => 5, "Crispy Seaweed" => 3}
```

Comment:
Metrics/LineLength: Line is too long. [142/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)